chingu-voyage4/Geckos-Team-32
# Development

## Purpose

- Merging Development into master to update Heroku app

## Approach/Fixes

- it'll update the Heroku master branch

## Testing

-  run `npm run build:prod`

- run `npm run serve`

- open app and make sure there are no strange breakpoints

- check to see you're still able to log in and youtube search works.
